### ISSUE TITLE
A good starter for reactjs & electorn-vite

### ISSUE THREAD

**Issue by alex8088** [daltonmenezes/electron-app#14]

**Comment by alex8088** [comment]
now, electorn-vite support source code protection 😄

**Comment by daltonmenezes** [comment]
Hi @alex8088 , thanks and thank you so much for the great work on Electron Vite! It's a really awesome and helpful project! 💜

I'm updating the Electron Vite in the template for the latest version.

About the Source Code Protection, should I set the `asar` to `false` in the `electron-builder.js` when bytecodePlugin is used?

**Comment by daltonmenezes** [comment]
@alex8088 also, `sandbox` should be `false` to getting this working, but the production build has the following warning `The vm module of Node.js is deprecated in the renderer process and will be removed.` when bytecodePlugin is used in the preload script. 🤔

**Comment by alex8088** [comment]
> @alex8088 also, `sandbox` should be `false` to getting this working, but the production build has the following warning `The vm module of Node.js is deprecated in the renderer process and will be removed.` when bytecodePlugin is used in the preload script. 🤔

Yes, I also noticed. Since the Electron 20 , the `sandbox` has also been set to true by default. We may need to give developers some tips.
